esp/src/eclwatch/QuerySetDetailsWidget.js

```diff
@@ -232,7 +232,7 @@ define([
                         tooltip += " " + newValue[i].Time;
                 }
                 this.graphsTab.set("tooltip", tooltip);
-            } else if (name === "ResourceURLCount" && newValue) {
+            } else if (name === "ResourceURLCount" && newValue - 1) {
                 this.widget._Resources.set("title", this.i18n.Resources + " (" + newValue + ")");
```

Comment:
I _think_ the bug is saying that the resource count is always 1 too many, in which case just subtract 1 from the displayed string as well as the test should fix it:
```
this.widget._Resources.set("title", this.i18n.Resources + " (" + (newValue - 1) + ")");
```